packages/client/src/v2-events/features/events/components/Pages.tsx

```diff
@@ -21,7 +21,12 @@ import {
 } from '@opencrvs/commons/client'
 import { MAIN_CONTENT_ANCHOR_ID } from '@opencrvs/components/lib/Frame/components/SkipToContent'
 import { FormFieldGenerator } from '@client/v2-events/components/forms/FormFieldGenerator'
-import { makeFormFieldIdFormikCompatible } from '@client/v2-events/components/forms/utils'
+import {
+  getDefaultValuesForFields,
```

Comment:
unused imports

Reply:
Ah yeah, this pr is still just a draft so havn't cleaned it up yet!